sentry-android-ndk/src/main/java/io/sentry/android/ndk/DebugImagesLoader.java

```diff
@@ -0,0 +1,89 @@
+package io.sentry.android.ndk;
+
+import io.sentry.SentryLevel;
+import io.sentry.SentryOptions;
+import io.sentry.android.core.IDebugImagesLoader;
+import io.sentry.protocol.DebugImage;
+import io.sentry.util.Objects;
+import java.util.Arrays;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.VisibleForTesting;
+
+final class DebugImagesLoader implements IDebugImagesLoader {
```

Comment:
`h`: If we remove the call to `options.isEnableNdk()` we could move this class to `io.sentry`.

Reply:
the reason to be in the Android module is that this is only necessary for the NDK integration, not normal Java apps.

Reply:
How do normal Java apps load `DebugImages`?

Reply:
they don't, normal Apps (non-native), don't require symbolication, stack traces are only obfuscated (via Proguard) for example.